fastapi_sqla/utils.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+import boto3
+
+ENDPOINT = os.getenv("RDS_ENDPOINT", "")
+REGION = os.getenv("AWS_REGION", "")
+USERNAME = os.getenv("USERNAME", "")
```

Comment:
```suggestion
USERNAME = os.getenv("SQLA_USERNAME", "")
```

Here I think we should use the same concept we did for GUESTLIST and use variables with prefix that are automatically loaded by the lib